wiki/Beatmap/Online_offset/zh.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - online song offset
+---
+
+# 在线偏移量
+
+*其他用途，请参见 [偏移量 (解疑)](/wiki/Disambiguation/Offset).*
+
+**在线偏移量** 是在谱面[ranked](/wiki/Beatmap/Category#ranked)后可以应用于谱面的偏移量。这通常是在[谱面](/wiki/Beatmap)需要微调的时候完成的。在线偏移量的优先级在玩家的全局偏移值和局部偏移值之上。
+
+它可以由提名评估团队的成员设置，并被一些高级用户看到，如[Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team)(NAT)和开发人员。这也意味着在线偏移量对网站上的普通用户是不可见的，但它会在游戏中显示出来。下面显示了在线偏移量如何显示给高级用户:
```

Comment:
missing two links

for reference:

`It can be set by members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), and be seen by some elevated users, such as [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) and [Developers](/wiki/People/The_Team/Global_Moderation_Team). This also means that online offset is invisible to normal users on the website, but it will always show in gameplay. Below shows how online offset is displayed to elevated users:`